_layouts/base.html

```diff
@@ -10,8 +10,8 @@
           <h2 class="nav-title">{{ site.title }}</h2>
         </a>
         <ul>
-          <li><a href="https://moekewlkatz.github.io/coffeeandteatime/">About</a></li>
-          <li><a href="https://moekewlkatz.github.io/Documentation-Portfolio/">Portfolio</a></li>
+          <li><a href="/coffeeandteatime/">About</a></li>
+          <li><a href="/coffeeandteatime/portfolio/">Portfolio</a></li>
```

Comment:
Starting the URL with "/" assumes the current base URL. Removing the base URL here will allow the links to still work when serving it locally.